.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+**Release note**:
+<!--  Steps to write your release note:
+1. Use the release-note-* labels to set the release note state (if you have access)
```

Comment:
Yes, you are right. So totally, we have "release-note" "release-note-none" and "release-note-needed". When people do release, just select all PRs with "release-note" label and you get every release notes. If we don't care about this PR in release just add "release-note-none" or leave "None". "needed" label is the way to enforce people to do this basically :)